package.json

```diff
@@ -8,54 +8,44 @@
     "lint": "tslint -p tsconfig.test.json",
     "prebuild": "rimraf index.js *.d.ts",
     "build": "tsc",
-    "prepretest": "rimraf ./coverage ./.nyc_output",
-    "pretest": "tsc -p tsconfig.test.json",
-    "test": "nyc ts-mocha ./test/test.js && npm run test-js",
-    "pretest-js": "npm run build && rimraf ./test-js-dir",
-    "test-js": "node test-js",
-    "posttest-js": "rimraf ./test-js-dir",
-    "posttest": "rimraf ./test",
-    "prepublishOnly": "npm run lint && npm run build"
+    "pretest": "rimraf ./coverage",
+    "test": "npm run lint && npm run jest",
```

Comment:
Mocha was giving me so much trouble, failing on Travis-CI builds for Node.js v6 and v8. I finally threw in the towel and switched to Jest, which seemed to solve all my problems!